CONTRIBUTING.rst

```diff
@@ -50,6 +50,17 @@ that black makes to your code after those changes are applied.
 
     $ pre-commit install
 
+Running tests
+^^^^^^^^^^^^^
+To run all of the tests (from the root directory of the repo):
+
+    $ pytest abcclassroom
```

Comment:
ok @kcranston looks like just running pytest works just fine!! that is the way our other packages are setup. so you can remove "abcclassroom"